translations/nl.json

```diff
@@ -110,21 +84,20 @@
     "new_notification": "Nieuwe melding"
   },
   "general": {
-    "describe_form": {
-      "title-separator": " · ",
-      "title-prefix-error": "Er ging iets fout!",
-      "pre-heading": "Stap {current} van {max}",
+    "form": {
+      "title_separator": " · ",
+      "title_prefix_error": "Er ging iets fout!",
+      "pre_heading": "Stap {current} van {max}",
       "back_button": "Vorige",
       "next_button": "Volgende",
       "submit_button": "Verstuur",
-      "not-required": "Dit veld is niet verplicht."
+      "required_short": "verplicht",
```

Comment:
Nu is er een general -> errors, is het dan niet fijn om de ```required_short``` en de ```not_required_short``` ook daarin te plaatsen in plaats van binnen in het form translatie object.

Reply:
ik had ze nu hierneer gezet omdat het niet echt foutmeldingen op zich zijn maar meer informatie over het formulier toch? tov bijvoorbeeld required en errors, die krijg je echt te zien op het moment wanneer er iets fout gaat